cypress/e2e/tests/rbac/review-assessment.test.ts

```diff
@@ -65,7 +65,7 @@ describe(["@tier2"], "Perform assessment and review as Architect", function () {
         cy.wait(10 * SEC);
         application[0].perform_assessment("medium", stakeholders);
         application[0].verifyStatus("assessment", "Completed");
-        application[0].validateAssessmentField("Medium");
+        // application[0].validateAssessmentField("Medium");
```

Comment:
Currently I don't see `Risk` section in right-side bar, waiting for dev's reply if that was done on purpose

Reply:
@ibragins ,Thanks for the PR . This is because of a known issue
[MTA bug](https://issues.redhat.com/browse/MTA-5883)

Reply:
Thanks @nachandr , added bug marker to the test